src/components/common/Extensions/UserHub/partials/BalanceTab/partials/PendingReputation/PendingReputation.tsx

```diff
@@ -6,7 +6,7 @@ import useUserReputation from '~hooks/useUserReputation.ts';
 import TimeRelative from '~shared/TimeRelative/index.ts';
 import TitleLabel from '~v5/shared/TitleLabel/index.ts';
 
-import reputationTabClasses from '../../ReputationTab.styles.ts';
+import reputationTabClasses from '../../BalanceTab.styles.ts';
```

Comment:
Nitpick, but should this object being imported be renamed to balanceTabClasses or similar? I see it's used in both Balance and Reputation things, so the file name change is a little confusing atm.

Reply:
of course it should!